did/.htaccess

```
@@ -2,5 +2,5 @@ Header set Access-Control-Allow-Origin *
 Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
 Options +FollowSymLinks
 RewriteEngine on
-RewriteRule ^$ https://w3c-ccg.github.io/did-spec/ [R=302,L]
-RewriteRule ^v(.+)$ https://w3c-ccg.github.io/did-spec/contexts/did-v$1.jsonld [R=302,L]
+RewriteRule ^$ https://www.w3.org/ns/did/ [R=302,L]
```

This link doesn't work.  Should it point at latest did-core spec?
```suggestion
RewriteRule ^$ https://w3c.github.io/did-core/ [R=302,L]
```

---

Good question. I'd forgotten that the W3C doesn't catch directory requests.

I'd just remove it...but it's up to y'all.